### DISCUSSION TITLE
Signoz vs ADOT in AWS

### DISCUSSION THREAD

**Discussion by seumassoltysik** [SigNoz/signoz#3843] [Q&A]
I am starting an investigation into using OTEL for my AWS infrastructure and applications and I am trying to understand the difference between using Signoz as the OTEL collector in AWS  vs using ADOT instead. Since ADOT was written by AWS is it better integrated into the AWS environment? Am I losing something if I use Signoz as opposed to ADOT?
Thanks.

**Comment by srikanthccv** [comment]
> Since ADOT was written by AWS is it better integrated into the AWS environment?

> Am I losing something if I use Signoz as opposed to ADOT?

You will not lose anything major by using SigNoz collector because most of the AWS components are already present in the upstream contrib, which we support, but it requires more work. You will find more examples and config templates for different environments and better support on ADOT repos than here. If your goal is to collect the telemetry data from the AWS environment and send them to SigNoz you can run ADOT and export to SigNoz using OTLP exporter; wherever the main SigNoz installation is running it should be SigNoz otel collector.

**Reply by seumassoltysik** [reply]
Thanks for the thorough answer.
One more question; If I am working only in AWS, what is the argument to adopt OTEL + Signoz as opposed to simply using the telemetry tools available in AWS(cloudwatch, cloudwatch metrics x-ray).

**Reply by srikanthccv** [reply]
Some to highlight off the top of my head 
- High-quality instrumentation with no vendor lock-in and a growing ecosystem
- The ability to receive data from multiple sources (receivers) and send them to multiple backends with little config changes
- Control over the amount and kind of data you send. Helps you reduce costs and noise
- An evolving first-class observability product that is native to OpenTelemetry
- The seamless co-relation experience in the same product without context switch b/w n number of tools

That being said, You can probably get by with Cloudwatch if it works fine for you but eventually, you may find it limiting in terms of capabilities, integrations etc.. and not having full control over your observability and tight lock-in.
